src/project/FileTreeView.js

```diff
@@ -538,7 +551,8 @@ define(function (require, exports, module) {
                     className: this.getClasses("jstree-leaf"),
                     onClick: this.handleClick,
                     onMouseDown: this.handleMouseDown,
-                    onDoubleClick: this.handleDoubleClick
+                    onDoubleClick: this.handleDoubleClick,
+                    onDragEnter: this.handleDragEnter
```

Comment:
it's interesting that we don't need to bind to `this` here even though we use `this` inside `handleDragEnter`. Maybe it does `.call(this, ...)`?